components/custom-network/network-information-step.tsx

```diff
@@ -41,7 +41,7 @@ export default function NetworkInformationStep({
     const name = e.target.value;
     const exists =
       name.trim() !== ""
-        ? name.toLowerCase().includes("bepro")
+        ? (name.toLowerCase().includes("bepro") || name.toLowerCase().includes("taikai"))
```

Comment:
```
['taikai', 'bepro'].includes(name.toLowerCase())
```

Reply:
for this case i need to know if the words "bepro" or "taikai" are in the name, the way you suggested we are going to verify if the name matches these words